src/gluonts/torch/distributions/distribution_output.py

```diff
@@ -106,13 +107,17 @@ class DistributionOutput(Output):
     """
 
     distr_cls: type
+    dim: int = 1
 
     @validated()
     def __init__(self) -> None:
         pass
 
     def _base_distribution(self, distr_args):
-        return self.distr_cls(*distr_args)
+        if self.dim == 1:
+            return self.distr_cls(*distr_args)
+        else:
+            return Independent(self.distr_cls(*distr_args), 1)
```

Comment:
Also I'm unsure about this mechanism: what if `self.distr_cls` returns a distribution that already has the correct `event_dim`, like a multivariate normal?

Reply:
ok i belieive i fixed that usecase now too